frontend/components/buttons/Button/_styles.scss

```diff
@@ -377,4 +377,11 @@ $base-class: "button";
     display: flex;
     justify-content: space-between;
   }
+
+  &--oversized {
+    background-color: $core-fleet-black;
+    padding: $pad-large $pad-small;
+    font-size: $medium;
+    width: 80%;
```

Comment:
This new variant looks good other than this specific width. Im thinking the 80% width isn't a sensible default. If anyone else were to use the oversize variant it would render by default of at 80% width or the parent. To me it seems this 80% width should be set in the parent component as the 80% seems specific to where it's being used atm, and we should set the default oversize width to 100%